src/components/common.js

```diff
@@ -597,3 +597,18 @@ export const PromptedConfirmationModal = ({ title, children, confirmationPrompt
     busy && spinnerOverlay
   ])
 }
+
+export const DeleteConfirmationModal = ({ objectName, objectType, children, ...props }) => {
```

Comment:
@nawatts can you change the DataTable `PromptedConfirmationModal` usage to call this new method?

Reply:
Will do.